### DISCUSSION TITLE
mqtt

### DISCUSSION THREAD

**Discussion by deanZZZZZ** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#654] [General]
v4.2.4
When I restart I have to re-enter the password for mqtt

**Comment by visualapproach** [comment]
Okay, just to be sure:
You saved the mqtt settings before restarting?
The mqtt will not connect until a few minutes after restart. Did you wait?
The password will not be visible on the mqtt config page even if it's saved.

How did you determine that the password has to be reentered?

**Reply by deanZZZZZ** [reply]
yes. I saved but I didn't wait 2 minutes :)
